authors/apps/comments/models.py

```diff
@@ -1,5 +1,6 @@
 from django.db import models
 
+from simple_history.models import HistoricalRecords
```

Comment:
@mwinel Please test out this pr and ensure it allows you to return unlimited number of edits and is not limited to only 4 or 5.

Reply:
Yes it does.